### PR TITLE
Add folder name in file manager

### PR DESCRIPTION
**Pull request by Hzj-jie**
A tiny change to resolve issue #2012.

### CODE REVIEW

File: frontend/apps/filemanager/filemanager.lua [hunk 1]
```diff
@@ -88,6 +92,11 @@ function FileManager:init()
     }
     self.file_chooser = file_chooser
 
+    function file_chooser:onPathChanged(path)  -- luacheck: ignore
+        FileManager.instance.banner[2].text = path
```
Comment (houqp) [inline frontend/apps/filemanager/filemanager.lua]: Sorry this is not documented, but it's better to use ` FileManager.instance.banner[2]:setText(path)` here.

Comment (Hzj-jie) [inline frontend/apps/filemanager/filemanager.lua]: Done. And I also added a CenterContainer to keep the text in the center.

**Comment by houqp** [comment]
Looks good to me :+1: Could you squash the commits into one?  I am going to bed soon so feel free to merge the PR after that :) Otherwise, I will merge it tomorrow.

**Comment by Hzj-jie** [comment]
Sorry, Huang merged this PR before I read your comment :)

**Comment by chrox** [comment]
It's OK in this case, both commits make sense in the title message.

**Comment by Hzj-jie** [comment]
Found an interesting issue, this change won't work on Kobo. I tested it on Linux emulator, and it works fine. Is there anything different between these two environments?

**Comment by Hzj-jie** [comment]
It looks like refresh strategies on Ubuntu and Kobo are not consistent. i.e. On ubuntu, the screen also refreshed. But on Kobo, only dirty rectangles are refreshed. So we should add a UIManager:setDirty(self, "partial") in TextWidget:setText function.

**Comment by houqp** [comment]
It's weird that it works in emulator without setDirty call. Emulator should only refresh dirty area too. I will take a look at this.

**Pull request merged**